extensions/Scribunto/engines/LuaCommon/lualib/ustring/ustring.lua

```diff
@@ -33,6 +33,9 @@ local function checkType( name, argidx, arg, expecttype, nilok )
 end
 
 local function checkString( name, s )
+    if type( s ) == 'number' then
```

Comment:
Minor: all these have inconsistent whitespace on the first line of the added blocks.